.github/workflows/bump-golang.yml

```diff
@@ -22,3 +25,5 @@ jobs:
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: ./.ci/bump-golang.yml
+          notifySlackChannel: "#ingest-notifications"
+          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@update-me-with-the-slack-team-to-be-poked` please look what's going on <${{ env.JOB_URL }}|here>"
```

Comment:
For instance, currently,  we are the ones receiving those notifications in Slack:

<img width="757" alt="image" src="https://github.com/elastic/fleet-server/assets/2871786/546fd0fb-e82f-4fd4-aacb-8b5bbd2c0e4a">

In this PR the ping will happen to `@update-me-with-the-slack-team-to-be-poked` - an unknown user/team in Slack. @jlind23, would you like to enable a specific ping in Slack? If so, what's the team name in Slack?